docs/faq.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: faq
-title: FAQ / Troubleshooting
+title: FAQ
```

Comment:
i am not sure why @gabro chose this title. perhaps faq is about "how does the formatter do something?" whereas this document also contains a lot of "why does the formatter do something?". it's possible that people who have this kind of question might not naturally select the faq page to look for an answer. just a theory.

perhaps the original author could help review.

Reply:
That particular title was migrated from the old website (see https://web.archive.org/web/20170320124508/https://scalameta.org/scalafmt) so the original author would be @olafurpg.
That said, I don't have any particular argument against the rename